planning/behavior_path_planner/src/behavior_path_planner_node.cpp

```diff
@@ -668,10 +709,12 @@ void BehaviorPathPlannerNode::onForceApproval(const PathChangeModule::ConstShare
 }
 void BehaviorPathPlannerNode::onMap(const HADMapBin::ConstSharedPtr msg)
 {
+  std::lock_guard<std::mutex> lock(mutex_);
   planner_data_->route_handler->setMap(*msg);
 }
 void BehaviorPathPlannerNode::onRoute(const HADMapRoute::ConstSharedPtr msg)
 {
+  std::lock_guard<std::mutex> lock(mutex_);
```

Comment:
In this function, the modified member variables are not only `planner_data_` but also `bt_manager_`, which are used in other functions as well.

So we have to do mutex for `bt_manager_` too. Also the performance would be better by using the other mutex like `mutex_bt_manager_`. But you can use the same mutex for `planner_data_` and `bt_manager_` if you want to. (I'm not sure how much the performance would be improved)

Reply:
Fixed at [e12262a](https://github.com/autowarefoundation/autoware.universe/pull/657/commits/e12262affa88455946c65b61ad4c0195a1fd8439).

`bt_manager_` is used only by initializer and `run()`